website/src/data/users.js

```diff
@@ -998,6 +998,14 @@ const Users = [
     source: null,
     tags: [ 'facebook'],
   },
+  {
+    title: 'Tinaël Devresse',
+    description: 'Personal website of Tinaël Devresse, a blog about anything that cross my mind which I think could interest y\'all.',
+    preview: require('./showcase/tinaeldevresse.png'),
+    website: 'https://tinaeldevress.eu',
+    source: 'https://github.com/hunteroi/tinaeldevresse.eu',
+    tags: ['personal', 'opensource', 'i18n']
```

Comment:
```suggestion
    tags: ['personal', 'opensource']
```

Does not seem to use i18n support for now